src/ProtocolFees.sol

```diff
@@ -21,12 +20,12 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
     mapping(Currency currency => uint256 amount) public protocolFeesAccrued;
 
     /// @inheritdoc IProtocolFees
-    IProtocolFeeController public protocolFeeController;
+    address public protocolFeeController;
 
     constructor() Owned(msg.sender) {}
 
     /// @inheritdoc IProtocolFees
-    function setProtocolFeeController(IProtocolFeeController controller) external onlyOwner {
+    function setProtocolFeeController(address controller) external onlyOwner {
         protocolFeeController = controller;
         emit ProtocolFeeControllerUpdated(address(controller));
```

Comment:
```suggestion
        emit ProtocolFeeControllerUpdated(controller);
```

Reply:
and same below in `setProtocolFee` and `collectProtocolFees` -> `address(protocolFeeController)`